pkg/kubelet/status/status_manager.go

```diff
@@ -184,7 +184,7 @@ func (m *manager) SetPodStatus(pod *v1.Pod, status v1.PodStatus) {
 
 	for _, c := range pod.Status.Conditions {
 		if !kubetypes.PodConditionByKubelet(c.Type) {
-			klog.Errorf("Kubelet is trying to update pod condition %q for pod %q. "+
+			klog.Infof("Kubelet is trying to update pod condition %q for pod %q. "+
```

Comment:
It sounds like this should be V(4), as it is expected behavior in some scenarios.

Reply:
@dashpole 
I have made the change.
Please take another look.